packages/yourloops/lib/auth/api.ts

```diff
@@ -552,18 +554,24 @@ async function logout(session: Readonly<Session>): Promise<void> {
   return Promise.reject(errorFromHttpStatus(response, log));
 }
 
+async function certifyProfessionalAccount(): Promise<IUser> {
```

Comment:
I am wondering if this actually useful to have this method here. All it does is call HttpService.post and returns its value. I can understand why it was done here before axios (as there was a lot of setup for http calls).
Maybe, it could be better (and easier) to have this code directly in auth/hook.tsx?
Do you have any opinion on this?